### DISCUSSION TITLE
Help to Importing Numeric Users-Only from Active Directory to Teampass

### DISCUSSION THREAD

**Discussion by m-blk** [nilsteampassnet/TeamPass#3852] [General]
Hi

I have encountered a challenge in importing numeric users, as illustrated in the attached screenshot.

![image](https://github.com/nilsteampassnet/TeamPass/assets/144053501/61897780-4e2d-4f5a-80d0-0b5d2a9ebd1e)

As depicted in the screenshot, your system is currently configured to import alphanumeric users successfully. However, i need to numeric-only users.

Here are some key details and requirements regarding the import process:

**Numeric Users-Only :** These are users with purely numeric usernames, and we need to import from the Active Directory.

**Challenges:** While your system is configured to handle alphanumeric usernames, it currently encounters issues when dealing with purely numeric usernames.

Thanks

**Comment by Leobaladf** [comment]
I have also this problem, my instalation was by docker-compose, but i have 2 problems.
- First, my users on active Directory are numerics and for this they dont be imported.
- Second problem, "this problem is in docker version and also hapening in ubuntu version", after a local user created i dont get change the category of user if he started administrator, dont is possible change for read-only ou a comum user.

**Comment by nilsteampassnet** [comment]
@marujoc 
OK I found the reason.
I will provide a fix for it
